libs/ui/search/src/lib/organisation-preview/organisation-preview.component.stories.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpClientModule } from '@angular/common/http'
+import { MatIconModule } from '@angular/material/icon'
+import { TranslateModule } from '@ngx-translate/core'
+import {
+  moduleMetadata,
+  Story,
+  Meta,
+  componentWrapperDecorator,
+} from '@storybook/angular'
+import { TRANSLATE_DEFAULT_CONFIG } from '@geonetwork-ui/util/i18n'
+import { RecordThumbnailComponent } from '../record-thumbnail/record-thumbnail.component'
+import { OrganisationPreviewComponent } from './organisation-preview.component'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
+
+export default {
+  title: 'Search/OrganisationPreviewComponent',
+  component: OrganisationPreviewComponent,
+  decorators: [
+    moduleMetadata({
+      declarations: [RecordThumbnailComponent],
+      imports: [
+        BrowserAnimationsModule,
+        HttpClientModule,
+        MatIconModule,
+        TranslateModule.forRoot(TRANSLATE_DEFAULT_CONFIG),
```

Comment:
Import `UtilSharedModule` as well (I get `The pipe 'safe' could not be found in the 'RecordThumbnailComponent'` error)